compaction/compaction_manager.cc

```diff
@@ -31,6 +31,8 @@
 #include "tombstone_gc-internals.hh"
 #include <cmath>
 #include "utils/labels.hh"
+#include "repair/incremental.hh"
```

Comment:
compaction manager shouldn't depend on repair, it's a layering violation.

compaction should expose its own notion of repair state, and repair should import it.